scriptmodules/emulators/np2pi.sh

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env bash
+
+# This file is part of The RetroPie Project
+#
+# The RetroPie Project is the legal property of its developers, whose names are
+# too numerous to list here. Please refer to the COPYRIGHT.md file distributed with this source.
+#
+# See the LICENSE.md file at the top-level directory of this distribution and
+# at https://raw.githubusercontent.com/RetroPie/RetroPie-Setup/master/LICENSE.md
+#
+
+rp_module_id="np2pi"
+rp_module_desc="NEC PC-9801 emulator"
+rp_module_help="ROM Extensions: .d88 .d98 .88d .98d .fdi .xdf .hdm .dup .2hd .tfd .hdi .thd .nhd .hdd .D88 .D98 .88D .98D .FDI .XDF .HDM .DUP .2HD .TFD .HDI .THD .NHD .HDD"
+rp_module_section="exp"
+
+function depends_np2pi() {
+    local depends=(libasound2-dev libsdl2-ttf-dev ttf-sazanami-gothic)
+    getDepends "${depends[@]}"
+}
+
+function sources_np2pi() {
+    git clone -b np2pi https://github.com/irori/SDL12-kms-dispmanx.git
+    cd SDL12-kms-dispmanx
+    git checkout d96f55822880ac26eecc8684ca44194b7220b1de
+    cd ..
+    git clone https://github.com/eagle0wl/np2pi.git
+    cd np2pi 
+    git checkout dabcc56e1fbf5f78b18a490810e93732681f36c7
```

Comment:
SDL12-kms-dispmanx is not needed - we have dispmanx included (add the dispmanx flag to module flags and use setDispmanx if it needs to be on by default - see other sdl1.2 emulators for examples.

why the specific commit ? if a specific commit is needed it needs a branch or tag on another fork.